content/posts/lets-code-contact-form-in-phoenix-part-one.md

```diff
@@ -266,4 +278,8 @@ defmodule MyApp.Support.MessageTest do
 end
 ```
 
-In the next post, I will be adding the module and functions for the `Support` context within the application. This new context will use the `Message` created in this post. Thanks for joining me! 👋
+In the next post, I will be adding the module and functions for the `Support` context within the application. This new context will use the `Message` and `changeset` function created in this post. Thanks for joining me! 👋
+
+## Acknowledgements
+
+I would like to thank [Troy Rosenberg](https://tmr08c.github.io/) and [Kate Studwell](http://katestud.github.io/) for their feedback on earlier drafts of this post. 🙇‍♂️🙏
```

Comment:
```suggestion
I would like to thank [Troy Rosenberg](https://tmr08c.github.io/) and [Kate Studwell](https://github.com/katestud) for their feedback on earlier drafts of this post. 🙇‍♂️🙏
```

Thanks so much for the shoutout! 
My github page is woefully outdated 😬 , so I'd suggest linking either to my profile or medium: https://medium.com/@katestudwell ❤️